### PR TITLE
Show moisture/flood alerts in the Area card

### PR DESCRIPTION
**Pull request by ghedo**
## Proposed change

This adds moisture/flood binary sensor values, alongside motion ones, to the Area card.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by zsarnett** [comment]
Can you share an image?

**Comment by ghedo** [comment]
@zsarnett this also includes #12977:

2022-06-21-131540_1001x574_scrot

File: src/panels/lovelace/cards/hui-area-card.ts [hunk 1]
```diff
@@ -72,6 +73,7 @@ const DOMAIN_ICONS = {
   sensor: { humidity: mdiWaterPercent },
   binary_sensor: {
     motion: mdiRun,
+    moisture: mdiWaves,
```
Comment (zsarnett) [inline src/panels/lovelace/cards/hui-area-card.ts:76]: I think `water-alert` would make more sense here

Comment (ghedo) [reply src/panels/lovelace/cards/hui-area-card.ts]: Makes sense, updated. This is what it looks like now:

2022-06-22-210121_910x525_scrot

**Review by zsarnett — APPROVED** [review]

**Pull request merged**